packages/voice/src/app/agent/page.tsx

```diff
@@ -202,14 +202,14 @@ const Button: React.FC<{ onClick: () => void; disabled: boolean; children: React
 const AgentPageComponent: React.FC = () => {
   const searchParams = useSearchParams();
   const agentId = searchParams.get('agent') || 'dr-donut';
-  const agentVoice = getAgent(agentId).ttsVoice;
+  const agentVoice = getAgent(agentId)?.ttsVoice;
   const tapOrClick = typeof window != 'undefined' && 'ontouchstart' in window ? 'Tap' : 'Click';
   const idleText = `${tapOrClick} anywhere to start!`;
   const asrProvider = searchParams.get('asr') || DEFAULT_ASR_PROVIDER;
   const asrLanguage = searchParams.get('asrLanguage') || undefined;
   const ttsProvider = searchParams.get('tts') || DEFAULT_TTS_PROVIDER;
   const ttsVoice = searchParams.get('ttsVoice') || agentVoice;
-  const model = searchParams.get('llm') || DEFAULT_LLM;
+  const model = getAgent(agentId) === undefined ? 'fixie' : searchParams.get('llm') || DEFAULT_LLM;
```

Comment:
Looks like we took slightly different approaches to identifying things as fixie, but feel free to stomp over my changes however you'd like. Main thing is just to ensure that speculative generation is disabled for fixie